### PR TITLE
Adding rx.scss, Bourbon, Neat, and variable partials.

### PR DESCRIPTION
**Pull request by webinista**

### CODE REVIEW

File: rx/assets/scss/rx.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import '../../../_sass/lib/bourbon/_bourbon.scss';
+@import '../../../_sass/lib/neat/_neat.scss';
+
+@import '../../../_sass/_va-variables.scss';
+@import '../../../_sass/core/_variables.scss';
+@import '../../../_sass/core/_grid-settings.scss';
+@import '../../../_sass/core/_grid.scss';
```
Comment (carlmjohnson) [inline rx/assets/scss/rx.scss]: Do you need to check in something (a Gemfile?) to make sure these libraries get installed? Looks like the build is failing.

Comment (carlmjohnson) [inline rx/assets/scss/rx.scss]: Looking at the Travis failure more closely reveals that it's complaining about JS, not SCSS, so it's probably unrelated.

Comment (webinista) [inline rx/assets/scss/rx.scss]: I need to check in `rx/public/js/generated/bundle.js` ... which I do not even see in my local copy.

**Comment by webinista** [comment]
Tests are failing because this file is not in the repo, but ... I can't find it in my local copy either. @alexose, can you help me out with this tomorrow?

``` 
- ./_site/rx/public/index.html
  *  internal script /public/js/generated/bundle.js does not exist (line 82)
  *  no favicon specified
```

**Pull request closed without merging**